### PR TITLE
Library

### PR DESCRIPTION
**Pull request by ThomasNagel**

### CODE REVIEW

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
 
             - name: simian check
               run: |
-                java -jar .github/simian-4.0.0.jar {balanceParentheses=true, failOnDuplication=true, ignoreCharacterCase=true, ignoreCurlyBraces=true, ignoreIdentifierCase=true, ignoreModifiers=true, ignoreStringCase=true, threshold=6} "backend/api/**" "backend/backend/**" "frontend/src/**" >> simian-out.txt || true
+                java -jar .github/simian-4.0.0.jar {balanceParentheses=true, failOnDuplication=true, ignoreCharacterCase=true, ignoreCurlyBraces=true, ignoreIdentifierCase=true, ignoreModifiers=true, ignoreStringCase=true, threshold=6} "runner.py" >> simian-out.txt || true
```
Comment (andreitudorp) [inline .github/workflows/quality.yml:55]: I think we should include here other files, not only the runner

File: .github/und_commands.txt [hunk 2]
```diff
@@ -1,7 +1,5 @@
 create -languages Python C++ web src_db.und
-add backend/api/
-add backend/backend/
-add frontend/src/
+add runner.py
```
Comment (andreitudorp) [inline .github/und_commands.txt:2]: same problem here

File: results/.gitkeep
Comment (andreitudorp) [inline results/.gitkeep:1]: what the sigma is this?

File: results/results.txt
Comment (andreitudorp) [inline results/results.txt:1]: Should results be in .gitignore?

File: settings.py
Comment (andreitudorp) [inline settings.py:1]: Not covered by Quality Check!

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+mkdir runenv
+
+cp -r submission/** /runenv
```
Comment (andreitudorp) [inline start.sh:3]: this copy operation will NOT work

File: results/.gitkeep
Comment (ThomasNagel) [reply results/.gitkeep:1]: this keeps the folder

**Review by andreitudorp — APPROVED** [review]
Looks good to me!
PolishToiletGIF

**Pull request merged**